packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/UserCell/UserCell.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { UserAvatar } from '@/components/UserAvatar/UserAvatar';
+import { useEntityUserById } from '@/core/queries';
+import { Chip, Skeleton } from '@mui/material';
+
+interface UserCellProps {
+  entityUserId: string;
+}
+
+export const UserCell = ({ entityUserId }: UserCellProps) => {
+  const { data: entityUser, isLoading } = useEntityUserById(entityUserId);
+
+  if (!entityUser) {
+    return null;
+  }
+
+  if (isLoading) {
+    return (
+      <Chip
+        label={
+          <Skeleton
+            variant="rounded"
+            height="50%"
+            width={100}
+            animation="wave"
+          />
+        }
+      />
+    );
+  }
+
+  const name = `${entityUser.first_name ?? ''} ${
+    entityUser.last_name ?? ''
+  }`.trim();
```

Comment:
Little simplification possible: 

`const name = (entityUser.first_name ?? '' + ' ' + entityUser.last_name ?? '').trim();
`
Suggesting here to just directly use the + operator for concatenation

Reply:
One more old fashion way:

```ts
const name = [entityUser.first_name, entityUser.last_name].filter(Boolean).join(' ') || undefined
```

Reply:
fixed